client/qiskit_serverless/core/clients/LocalClient.py

```diff
@@ -0,0 +1,230 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+================================================
+Provider (:mod:`qiskit_serverless.core.client`)
+================================================
+
+.. currentmodule:: qiskit_serverless.core.client
+
+Qiskit Serverless provider
+===========================
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ComputeResource
+    ServerlessClient
+"""
+import json
+import logging
+import os.path
+import os
+import re
+import sys
+from typing import Optional, List, Dict, Any, Union
+from uuid import uuid4
+
+import subprocess
+from subprocess import Popen
+
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+from qiskit_serverless.core.constants import (
+    OT_PROGRAM_NAME,
+    ENV_JOB_ARGUMENTS,
+)
+from qiskit_serverless.core.client import BaseClient
+from qiskit_serverless.core.job import (
+    Job,
+    Configuration,
+)
+from qiskit_serverless.core.function import QiskitFunction
+from qiskit_serverless.exception import QiskitServerlessException
+from qiskit_serverless.serializers.program_serializers import (
+    QiskitObjectsEncoder,
+)
+
+TIMEOUT = 30
```

Comment:
Correct me if I'm wrong but this `TIMEOUT` is not being used (I didn't find any reference at least) and, in any case, I think we should use `REQUESTS_TIMEOUT` in `constants` probably.

Reply:
Sound like a bad copy and paste :)